src/avt/Filters/avtXRayFilter.C

```diff
@@ -682,8 +774,12 @@ avtXRayFilter::Execute(void)
 
         if (cellDataType == VTK_FLOAT)
             ImageStripExecute<float>(totalNodes, dataSets);
-        else // if (cellDataType == VTK_DOUBLE)
+        else if (cellDataType == VTK_DOUBLE)
             ImageStripExecute<double>(totalNodes, dataSets);
+        else
+            {
+                // TODO: else error
```

Comment:
Are you gonna put an exception in here?

Reply:
good catch, it wasn't there before, I wanted to add an error. 

- [ ] cyrus fix this

Reply:
looked at this more, throwing an error successful here would be quite hard given the structure of the code (allocations, etc). 

There are also many other cases where only float and double are checked w/o errors.  I am going to skip adding this case.